_maps/metastation.dm

```diff
@@ -17,7 +17,7 @@ z7 = empty space
 	#include "map_files\MetaStation\MetaStation.v41A.II.dmm"
 	#include "map_files\MetaStation\z2.dmm"
 	#include "map_files\generic\tcommsat-blown.dmm"
-	#include "map_files\MetaStation\z4.dmm"
+	#include "map_files\generic\z4.dmm"
 	#include "map_files\generic\Lavaland.dmm"
 	#include "map_files\generic\z6.dmm"
 	#include "map_files\generic\z7.dmm"
```

Comment:
Meta's Z4 is the cyberiad Z6 from 4 years ago, not the engineering outpost.

This seems unintentional unless youre also killing that, which if so, I would add that to the PR description. 

Also if this map is no longer used, shouldn't it be removed from the files and `travis_map_testing.dm`

Reply:
Oh, not intentional, but might as well.